src/components/connect.js

```diff
@@ -208,13 +190,22 @@ export default function connect(mapStateToProps, mapDispatchToProps, mergeProps,
         }
       }
 
-      componentDidMount() {
+      componentWillMount() {
```

Comment:
Used WillMount instead of DidMount because now on the first render we have to trigger `handleChange` otherwise mergedProps don't get calculated. Is it a problem?